README.md

```diff
@@ -84,15 +84,16 @@ The current state presents a foundation that our research projects [AGRARSENSE](
 WayWiseR is divided into several ROS2 packages. Make sure to have a look into the respecitve package.xml files.
 
 - **waywiser**: A meta package that depends on all packages below to be able to refer to WayWiseR as a whole.
-- **waywiser_twist_safety**: Contains configuration and launch files to manage twist commands to vehicle from different sources. It also includes a composable node for emergency stop monitoring.
 - **waywiser_description**: Contains vehicle descriptions in the form of [xacro](https://docs.ros.org/en/humble/Tutorials/Intermediate/URDF/Using-Xacro-to-Clean-Up-a-URDF-File.html) files. Currently a single vehicle is available that corresponds to a [Traxxas](https://traxxas.com/) Slash incl. camera, depth camera, LiDAR and IMU.
 - **waywiser_gazebo**: Everything related to simulation using [Gazebo](https://gazebosim.org).
 - **waywiser_hwbringup**: Configuration and launch files to get real (not simulated) vehicles running.
+- **waywiser_nav2**: Package that provides dynamic path planning using Nav2.
```

Comment:
```suggestion
- **waywiser_nav2**: Package that provides dynamic path planning using [Nav2](https://navigation.ros.org/).
```

Reply:
Updated it in https://github.com/RISE-Dependable-Transport-Systems/WayWiseR/pull/37/commits/1bd47555f0c87186f0f8212eeac61ee1e2ffe19a